main.py

```diff
@@ -17,58 +18,36 @@ def selfie():
 
 @app.route("/recipies", methods=["GET"])
 def recipies():
-    food = [
-        "Ewedu Soup = ewedu_soup",
-        "African Salad - Abacha and Ugba = african_salad",
-        "Tuwo Shinkafa = tuwo_shinkafa",
-        "Classic Hamburger = classic_hamburger", "Pizza = pizza",
-        "Pancakes and Scrambled Eggs = pancakes_and_scrambled_eggs",
-        "Shawarma = shawarma", "Small Chops = small_chops"
-    ]
-    with open("files/cippe.json", "r") as items:
-        return jsonify({"status": "success", "data": json.load(items)})
+    cippie_db = sqlite3.connect("cippe.db")
+    cursor = cippie_db.cursor()
+    recipe = cursor.execute("SELECT food_name FROM recipies")
+    return jsonify({"status": "success", "data": recipe.fetchall()})
 
 
 @app.route("/recipies/<name>", methods=["GET"])
 def get_recipie(name):
-    with open("files/{}.json".format(name), "r") as food:
-        return jsonify({"status": "success", "data": json.load(food)})
+    cippie_db = sqlite3.connect("cippe.db")
+    cursor = cippie_db.cursor()
+    recipies = cursor.execute(
+        "SELECT food_name, description, top_image, bottom_image FROM recipies WHERE food_name=?",
+        (name, ))
+    return jsonify({"status": "success", "data": recipies.fetchall()[0]})
```

Comment:
```suggestion
        "SELECT food_name, description, top_image, bottom_image FROM recipies WHERE food_name=? ORDER BY food_name ASC LIMIT 1", (name, ))
    return jsonify({"status": "success", "data": recipies.fetchall()})
```
You can get the first one like this